T3/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Homework_Task1_3
+{
+	internal class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.WriteLine("Enter the numbers of the array:");
+			int num=int.Parse(Console.ReadLine());
+
+			int[] array = new int[num];
+
+			for(int i=0; i<num; i++)
+			{
+				Console.WriteLine($"Enter {i + 1} element: ");
+				array[i] = int.Parse(Console.ReadLine());
+			}
+
+            Console.Write("Original array: ");
+			PrintArray(array);
+
+			int [] newArray =SortedArray(array);
+            Console.WriteLine(	);
+            Console.Write("New array based on the principle Each Value - Only Once: ");
+			PrintArray( newArray );
+			Console.ReadKey();
+        }
+		static int [] SortedArray(int[] array)
```

Comment:
1. Why SortedArray? You need to make unique values in an array... At least bad name
2.  Not clear logic, three enclosed loops too complicated and not efficient for the task, think how to simplify. If necessary ask me for help)))